beacon-chain/blockchain/process_block.go

```diff
@@ -327,6 +327,14 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.ReadOnlySi
 	return s.saveHeadNoDB(ctx, lastB, lastBR, preState)
 }
 
+func updateNextSlotCacheOnBlock(root [32]byte, st state.BeaconState) {
+	slotCtx, cancel := context.WithTimeout(context.Background(), 2*slotDeadline)
```

Comment:
slotDeadline is changed to x2 versus before. If there's no reason, we should revert